packages/salesforcedx-utils-vscode/src/helpers/paths.ts

```diff
@@ -132,8 +129,13 @@ function debugLogsFolder(): string {
   return pathToDebugLogsFolder;
 }
 
+function sfdxProjectConfig(): string {
+  const pathToSFDXProjectConfig = path.join(projectPaths.stateFolder(), SFDX_CONFIG_FILE);
+  return pathToSFDXProjectConfig;
+}
+
 function toolsFolder(): string {
-  const pathToToolsFolder = path.join(stateFolder(), TOOLS);
+  const pathToToolsFolder = path.join(projectPaths.stateFolder(), TOOLS);
```

Comment:
@gbockus-sf here's that change, which did fix the test - I updated the other spots using stateFolder too so the path to test those is easier